products/styles.css

```diff
@@ -0,0 +1,133 @@
+.products-container {
+  padding: 2.5rem 5rem;
+  background: #f7f7f7;
+  flex: 1;
+  height: calc(100vh - 10rem);
+  overflow-y: auto;
+}
+.main-container {
+  display: flex;
+}
+.product-sidebar {
+  width: 280px;
+  height: calc(100vh - 10rem);
+  overflow-y: auto;
+  background: #fff;
+  padding: 2.5rem;
+}
+
+.sidebar-title {
+  display: flex;
+  width: 100%;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 1.5rem;
+  margin-bottom: 2.5rem;
+  justify-content: space-between;
+}
+.sidebar-title .clear-option {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1.5rem;
+  text-decoration-line: underline;
+}
+
+.sidebar-price-filter {
+  display: flex;
+  flex-direction: column;
```

Comment:
you can extract this in a separate utility class and then use it everywhere to avoid repeating the same set of code